palindrome of intiger

```diff
@@ -0,0 +1,29 @@
+#include <bits/stdc++.h>
```

Comment:
Change file name. There is spelling mistake and don't include spaces in file names